src/transformNestedSelection.ts

```diff
@@ -168,22 +181,25 @@ const mergeFieldNodesAndWrapper = (fieldNodesDoc: DocumentNode, wrapper: Documen
 }
 
 const transformNestedSelection = (
+  schema: GraphQLSchema,
   info: GraphQLResolveInfo,
   prefix: string,
   wrapper?: DocumentNode,
 ) => {
   if (!wrapper) {
     const infoDoc = transformInfoIntoDoc(info)
     const {variables, document: prefixedDoc} = pruneUnused(infoDoc, info.variableValues)
-    const document = unprefixTypes(prefixedDoc, prefix)
+    const unprefixedDocument = unprefixTypes(prefixedDoc, prefix)
+    const document = pruneExtendedFields(schema, unprefixedDocument)
     return {document, variables, wrappedPath: undefined}
   }
   const fieldNodesDoc = transformInfoIntoDoc(info)
   const fieldNodesWithoutLocalInterfacesDoc = pruneInterfaces(fieldNodesDoc, prefix, info)
   const unprefixedFieldNodesDoc = unprefixTypes(fieldNodesWithoutLocalInterfacesDoc, prefix)
+  const fieldNodesWithoutExtendedFields = pruneExtendedFields(schema, unprefixedFieldNodesDoc)
```

Comment:
I did not test this branch, so I'm not 100% sure it's correct. Help would be appreciated.

Reply:
https://github.com/ParabolInc/parabol/pull/6026/files will test this branch IIRC because it accesses the integration via the `Task {integration}` instead of the `api {data}`